sdk/communication/Azure.Communication.Common/src/ThreadSafeRefreshableAccessTokenCache.cs

```diff
@@ -210,7 +210,7 @@ public ScheduledAction(Action action, TimeSpan period)
             public void Dispose()
                 => _timer.Dispose();
 
-            private void OnTimerTick(object _)
+            private void OnTimerTick(object? _)
```

Comment:
nullable is enabled in this file: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/communication/Azure.Communication.Common/src/ThreadSafeRefreshableAccessTokenCache.cs#L10